src/pages/item/index.js

```diff
@@ -1,12 +1,59 @@
-import React from 'react';
+import React, { useContext, useState } from 'react';
+import '../../App.css'
+import './item.css'
+import {ItemContext} from '../../context/items'
+import {BasketContext} from '../../context/basket'
+import ItemList from '../home/items-list'
+import _ from 'lodash'
+import {useParams} from 'react-router-dom'
+import ItemTemplate from '../../components/item-template/item-template'
+
+
+
+ 
+
+const Item = (props) => {
+
+  const basket = useContext(BasketContext)
```

Comment:
use destructuring ie {updateBasket} = useContext(BasketContext). no need to get all the values if we only use the one function